### PR TITLE
hotfix for loading titles, the api was buggy (not strict)

### PR DESCRIPTION
**Pull request by yashha**

### CODE REVIEW

**Comment by mammuth** [comment]
The fix seems not to work for eg. "Aegon Targaryen (son of Rhaegar)"

**Comment by yashha** [comment]
Another problem is thoros of myr has no plod so everything fails when he is displayed
https://api.got.show/api/plod/bySlug/Thoros_of_Myr

**Comment by yashha** [comment]
Aegon Targaryen (son of Rhaegar) has only gotarffplod
https://api.got.show/api/plod/bySlug/Aegon_Targaryen_(son_of_Rhaegar)

**Comment by gyachdav** [comment]
Need to check with @goldbergtatyana how come he doesn't have a plod. In the meantime just comment out the problematic ones.

**Comment by yashha** [comment]
Really strange, here he has gotplod: 
https://api.got.show/api/plod/bySlug/Aegon_Targaryen

**Comment by gyachdav** [comment]
Or put some logic that would overcome cases of characters without PLODs

**Comment by yashha** [comment]
Shell I make them have plod of 0%?
Thats strange too

**Comment by gyachdav** [comment]
That's a different Aegon :)

**Comment by gyachdav** [comment]
No no no. Just randomize anothe pair.

**Comment by gyachdav** [comment]
Or comment the problematics out. It's the same thing.

**Comment by yashha** [comment]
I would rather comment it out until their fixed :)

**Comment by goldbergtatyana** [comment]
@yashha @gyachdav 
Aegon Targaryen (son of Rhaegar) is dead, so we should not care about his PLODs anyways!

However, if looking what both groups return for him, I find 
- gotplod (Group 6) = 0.997 https://github.com/Rostlab/JS16_ProjectB_Group6/files/190822/weka_pred.json.txt
- gotarffplod (Group 7) = 0.826 (dont know where their json file is, but here are their raw predictions https://rostlab.org/~goldberg/NaiveBayes.txt)

**Comment by goldbergtatyana** [comment]
I checked if there are characters with plod predictions by one group, but not the other. I found that group 6 (gotplod) has predictions for seven characters, for which group 7 (gotarffplod) does not have predictions. Six of them are:
Cohollo
Doreah
Drogo
Haggo
Qotho
Rhaego

However **attention** all these six characters are dead, so we should never look at their plods.

The seventh character predicted by group 6 (gotplod) only is **Henly (House Ashford)**, but I cannot find him in the characters section of got.show. So, we should not care about him either.

There are no characters predicted by group 7, but not by grop 6.

Long story short, if a character is alive then we should have predictions of both _gotplod _and _gotarffplod _for him. If this is not the case, then it is a bug and need to be inspected.
